apps/web/src/utils/logger.ts

```diff
@@ -0,0 +1,5 @@
+import { getTimestamp } from '~/shared/utils/time';
+
+export const _log: Console['log'] = (...args) => {
+  return console.log(`%c[${getTimestamp()}]`, 'color:#4878fa', `${args[0]}`, ...args.slice(1));
```

Comment:
cool hack to add color in the browser

<img width="369" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/d59be9da-debe-4360-9264-307fda36edf9">

Reply:
woah